less/admin/PermissionsPage.less

```diff
@@ -49,19 +50,36 @@
     color: @muted-color;
   }
   thead th {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0;
+    padding-bottom: 10px;
```

Comment:
I don't think this is necessary 🤔

Reply:
I didn't find another way to achieve the main point of this idea, i.e. stick the table heads. I don't know if `-webkit-sticky` is necessary as I don't really understand the safari world, but concerning the others, `position` and `top` are necessary to stick the columns headers, and `padding-bottom` is here to correct a side effect of them (the default `padding: 5px` seems very poor when scrolling down).

Maybe you're just suggesting to not stick the `thead th` and only stick the `tbody th`?

Reply:
my apologies, I was only referring to the bottom padding but for perhaps mistakenly selected all lines or smth. There is nothing wrong with the rest.

> and padding-bottom is here to correct a side effect of them (the default padding: 5px seems very poor when scrolling down)

makes sense 👍🏼

Reply:
OK no problem!